tests/categorical_test.py

```diff
@@ -112,7 +112,26 @@ def testBinop(self):
         self.assertEqual(('type of argument "other" must be one of (Categorical, str, str_);' +
                           ' got int instead'), 
                          cm.exception.args[0])
-            
+    
+    def testIn1d(self):
+        vals = [i for i in range(0, 5)]
```

Comment:
The test vectors should have some duplicates, in order to truly exercise the `Categorical` functionality. For example,

```python
vals = [i % 3 for i in range(10)]
valsTwo = [i % 2 for i in range(10)]
# Stringify and Categoricalify
...
answer = ak.array([x < 2 for x in vals])
self.assertTrue((answer == ak.in1d(catOne, catTwo)).all())
```

Side note: the unit tests should reflect the preferred idiom of `ak.in1d(catOne, catTwo)`, as opposed to the method call `catOne.in1d(catTwo)`. When I originally wrote the method, I probably should have prefixed it with an underscore, because I did not intend for users to call it directly.

Reply:
@reuster986 ready for review